indico/modules/categories/util.py

```diff
@@ -71,6 +71,16 @@ def get_contribs_by_year(category_id=None):
     return OrderedDict(query)
 
 
+def get_min_year(category_id=None):
+    """Get the min year.
+
+    :param category_id: The category ID to get statistics for..
+    :return: A `Datetime Object`.
```

Comment:
A `datetime` object or something similar

Reply:
why something similar, it will always be a `Dateime object`

Reply:
it's `datetime`, not `Datetime` ;)

Reply:
Also, the `Object ` part shouldn't be inside the backticks

Reply:
btw, there's a `.` too much in the line before.

Reply:
So the return value is of type `datetime` not `Datetime` and IMO what is between '`' refers to the actual type. `Datetime Object` is not a type.

Reply:
okay, got it. changed :+1: